src/main/java/core/basesyntax/service/WriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterService {
+    void writeToFile(String fileName, String report);
```

Comment:
```suggestion
    void writeToFile(String fileName, String data);
```
this service can write not only report